crocks/List.spec.js

```diff
@@ -119,6 +120,14 @@ test('List value', t => {
   t.end()
 })
 
+test('List toArray', t => {
+  const a = List([ 'some-thing', ['else', 43], 34 ]).toArray()
+
+  t.same(a, [ 'some-thing', 'else', 43, 34 ], 'folds the wrapped array')
+
+  t.end()
+})
```

Comment:
I did not know whether to flatten the array so I figured not, but perhaps a `toFlatArray` (or something better-efficaciously named) could do that.

And for the `toFlatArray`, would this fly?
```
const emojo = List([ 'some-thing', ['else', [43]], 34])
t.same.(emojo, ['some-thing', 'else', 43, 34]`, 'flatten and folds wrapped array'
t.end()
```